client/platform/desktop/backend/serializers/cli.ts

```diff
@@ -5,26 +5,60 @@
  * Command-line entrypoints into serializers and other tooling.
  * See README.md for usage
  */
+import { Writable } from 'stream';
+import { stdout } from 'process';
 
 import yargs from 'yargs';
 
-import { parseFile } from './viame';
+// Improper relative paths used because CLI won't work with path aliases
+// https://github.com/TypeStrong/ts-node/issues/422
+import { loadJsonTracks, loadJsonMetadata } from 'platform/desktop/backend/native/common';
+import { parseFile, serialize } from './viame';
+
+// https://stackoverflow.com/questions/21491567/how-to-implement-a-writable-stream
+function echoStream() {
+  return new Writable({
+    write(chunk, encoding, next) {
+      stdout.write(chunk.toString());
+      next();
+    },
+  });
+}
 
 async function parseViameFile(file: string) {
   const tracks = await parseFile(file);
-  // eslint-disable-next-line no-console
-  console.log(JSON.stringify(tracks));
+  stdout.write(JSON.stringify(tracks));
+}
+
+async function parseJsonFile(filepath: string, metapath: string) {
+  await Promise.all([
+    loadJsonTracks(filepath),
+    loadJsonMetadata(metapath),
+  ]).then(([input, meta]) => serialize(echoStream(), input, meta));
 }
 
 const { argv } = yargs
-  .command('viame [file]', 'Parse VIAME CSV', (y) => {
+  .command('viame2json [file]', 'Convert VIAME CSV to JSON', (y) => {
+    y.positional('file', {
+      description: 'The file to parse',
+      type: 'string',
+    }).demandOption('file');
+  })
+  .command('json2viame [file] [meta]', 'Convert JSON to VIAME CSV', (y) => {
     y.positional('file', {
       description: 'The file to parse',
       type: 'string',
-    });
+    }).demandOption('file');
+    y.positional('meta', {
+      description: 'The metadata to parse',
+      type: 'string',
+    }).demandOption('meta');
```

Comment:
Question:  Confirming that this limits the cli converter to only be used on Desktop generated JSON for right now?  Obviously any download JSON from the web wouldn't convert because of this requirement.  The meta includes the `fps`, `confidenceFilters` and the `imagenames` used for exporting.  This things could be removed to enable support but I believe we want to have image names/video frame timestamps in all outputs anyways.

Reply:
CLI is, so far, only used by us developers to test and debug serializers.  the information from meta.json is always required.  I'm not really interested in coming up with some other means of supplying that data to the serializer, such as implementing directory crawl, girder integration, etc.